src/components/Search/FilterDropdowns/UserSelectPopup.tsx

```diff
@@ -114,7 +113,8 @@ function UserSelectPopup({value, closeOverlay, onChange}: UserSelectPopupProps)
                     return 1;
                 }
                 return 0;
-            });
+            })
+            .filter((option) => option.accountID !== accountID);
```

Comment:
This filter shouldn't be required. The personal details shouldn't have the current user included.